src/components/QueryBuilder.vue

```diff
@@ -22,9 +25,25 @@
 <script lang="ts">
 import Vue from 'vue';
 import TextInput from '@wmde/wikit-vue-components/src/components/TextInput.vue';
+import { mapState } from 'vuex';
 
 export default Vue.extend( {
 	name: 'QueryBuilder',
+
+	data() {
+		return {
+			textInputValue: '',
+			textInputProperty: ''
```

Comment:
Neither of these are actually used. I think the `textInputProperty` one can just be deleted, and the value TextInput should actually read from the vuex store in the same way that the property one does, i.e. mapped it to a computed property from the store state.